src/Stryker.Core/Stryker.Core.UnitTest/Mutants/MutantOrchestratorTests.cs

```diff
@@ -205,6 +205,15 @@ public void ShouldNotMutateConst()
             ShouldMutateSourceToExpected(source, expected);
         }
 
+        [Fact]
+        public void ShouldNotMutateEnum()
```

Comment:
Please add a message as to why enums should not be mutated. We could forget why this is necessary.